src/lib/effects.ts

```diff
@@ -1,14 +1,24 @@
 import { Effect, toSkill } from "kolmafia";
-import { $effects, have } from "libram";
+import { $effects, have, $familiar, $effect } from "libram";
+
+import { runsOrNCFamiliar } from "./familiar";
 
 function filterHave(effects: Effect[]) {
   return effects.filter((effect) => have(toSkill(effect)));
 }
 
 export function basicEffects(): Effect[] {
-  return filterHave(
-    $effects`Leash of Linguini, Empathy, Blood Bond, Springy Fusilli, Cletus's Canticle of Celerity`,
-  );
+  const buffs: Effect[] = [
+    $effect`Leash of Linguini`,
+    $effect`Empathy`,
+    $effect`Blood Bond`,
+    $effect`Springy Fusilli`,
+    $effect`Cletus's Canticle of Celerity`,
+  ];
+  if (runsOrNCFamiliar() === $familiar`Frumious Bandersnatch`) {
```

Comment:
Will need to wrap the calls to `basicEffects` in a closure.

For example, in PLD we have
```
        effects: [...basicEffects(), ...noncombatEffects()],
```
this is evaluated once when the task is initialized. We will either always have ode, or never have bode as a basic effect.

`effects: () => [...basicEffects(), ...nonCombatEffects()]` is what we want, though my syntax may be wrong.